### PR TITLE
Adding a prometheus metric for submit queue delay

### PR DESCRIPTION
**Pull request by Q-Lee**
We should monitoring the time since the last submit.

https://prometheus.io/docs/practices/instrumentation/#timestamps,-not-time-since
For reasons, I've killed the PR in #3164

### CODE REVIEW

File: mungegithub/mungers/submit-queue.go [hunk 1]
```diff
@@ -201,6 +202,10 @@ var (
 			Name: "submitqueue_merge_total",
 			Help: "Number of merges done",
 		}),
+		MergeDelay: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "submitqueue_merge_delay",
+			Help: "Time since last merge",
```
Comment (rmmh) [inline mungegithub/mungers/submit-queue.go]: Should be "Time of last merge"-- and maybe s/MergeDelay/LastMergeTime/?

Comment (Q-Lee) [reply mungegithub/mungers/submit-queue.go]: Fair enough.

File: mungegithub/mungers/submit-queue.go [hunk 2]
```diff
@@ -200,6 +201,10 @@ var (
 			Name: "submitqueue_merge_total",
 			Help: "Number of merges done",
 		}),
+		LastMergeTime: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "submitqueue_merge_delay",
```
Comment (rmmh) [inline mungegithub/mungers/submit-queue.go]: Name is wrong :)

File: mungegithub/mungers/submit-queue.go [hunk 3]
```diff
@@ -545,6 +551,7 @@ func (sq *SubmitQueue) EachLoop() error {
 	sq.updateHealth()
 	sqPromMetrics.OpenPRs.Set(float64(len(sq.prStatus)))
 	sqPromMetrics.QueuedPRs.Set(float64(len(sq.githubE2EQueue)))
+	sqPromMetrics.LastMergeTime.Set(float64(sq.lastMergeTime.Unix()))
```
Comment (rmmh) [inline mungegithub/mungers/submit-queue.go]: Put this in updateMergeRate() instead?

File: mungegithub/mungers/submit-queue.go (hunk 2 above)
Comment (Q-Lee) [reply mungegithub/mungers/submit-queue.go]: lol

File: mungegithub/mungers/submit-queue.go [hunk 4]
```diff
@@ -200,6 +201,10 @@ var (
 			Name: "submitqueue_merge_total",
 			Help: "Number of merges done",
 		}),
+		LastMergeTime: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "submitqueue_time_of_last_merge",
```
Comment (rmmh) [inline mungegithub/mungers/submit-queue.go]: "time_of_last_merge" vs "last_merge_time" ?

**Pull request merged**